pom.xml

```diff
@@ -234,7 +234,7 @@
     </dependencyManagement>
 
     <properties>
-        <org.wso2.carbon.identity.framework.version>5.16.13</org.wso2.carbon.identity.framework.version>
+        <org.wso2.carbon.identity.framework.version>5.12.153</org.wso2.carbon.identity.framework.version>
```

Comment:
this should point to the latest framework version which will get merged once https://github.com/wso2/carbon-identity-framework/pull/2703 gets merged